.nox/design/store.entity.nox.yaml

```diff
@@ -25,15 +26,20 @@ attributes:
       maxLength: 63
     isRequired: true
 
-  - name: PhysicalMoney
-    description: Physical Money in the Physical Store
-    type: money
-    moneyTypeOptions:
-      defaultCurrency: GBP      
-      decimalDigits: 5
-      integerDigits: 10
+  - name: Address
+    description: Store address
+    type: streetAddress
+    isRequired: true
+
+  - name: LatLong
+    description: Store location coordinates
+    type: latLong
+    isRequired: true
+
+  - name: Phone
```

Comment:
I thought we would drop "Store" and replace with "VendingMachine". It may have a "SupportNumber" but not a "Phone"

Reply:
Renamed Store entity to VendingMachine